reactor-core/build.gradle

```diff
@@ -57,90 +57,89 @@ configurations {
 
 dependencies {
 	// Reactive Streams
-	api "org.reactivestreams:reactive-streams:${reactiveStreamsVersion}"
-	tckTestImplementation ("org.reactivestreams:reactive-streams-tck:${reactiveStreamsVersion}") {
+	api libs.reactiveStreams
+	tckTestImplementation (libs.reactiveStreams.tck) {
 		// without this exclusion, testng brings an old version of junit which *embeds* an old version of hamcrest
 		// which gets picked up first and that we don't want. TCK runs fine w/o (old) junit 4.
 		exclude group: 'junit', module: 'junit'
 	}
+	tckTestImplementation libs.testNg
```

Comment:
the exclusion above should probably be applied there instead, if that import is indeed needed